bot/exts/moderation/silence.py

```diff
@@ -93,87 +93,222 @@ async def _async_init(self) -> None:
         await self.bot.wait_until_guild_available()
 
         guild = self.bot.get_guild(Guild.id)
-        self._verified_role = guild.get_role(Roles.verified)
+
+        self._verified_msg_role = guild.get_role(Roles.verified)
+        self._verified_voice_role = guild.get_role(Roles.voice_verified)
+        self._helper_role = guild.get_role(Roles.helpers)
+
         self._mod_alerts_channel = self.bot.get_channel(Channels.mod_alerts)
+
         self.notifier = SilenceNotifier(self.bot.get_channel(Channels.mod_log))
         await self._reschedule()
 
+    async def _get_related_text_channel(self, channel: VoiceChannel) -> Optional[TextChannel]:
+        """Returns the text channel related to a voice channel."""
+        # TODO: Figure out a dynamic way of doing this
+        channels = {
+            "off-topic": Channels.voice_chat,
+            "code/help 1": Channels.code_help_voice,
+            "code/help 2": Channels.code_help_voice,
+            "admin": Channels.admins_voice,
+            "staff": Channels.staff_voice
+        }
+        for name in channels.keys():
+            if name in channel.name.lower():
+                return self.bot.get_channel(channels[name])
+
+    async def send_message(self, message: str, source_channel: TextChannel,
+                           target_channel: Union[TextChannel, VoiceChannel],
+                           alert_target: bool = False, duration: HushDurationConverter = 0) -> None:
```

Comment:
I hadn't seen that guide before, thank you. I'll fix the rest of the style issues.

Reply:
It's still incorrect. Every argument must be on a separate line You also inconsistently used 4 and 8 space indents. There is also some test code which has similar style issues. For long function calls, consider assigning an argument to a variable and then passing the variable to the function. This will avoid having to break up the function into multiple lines. For example,

```py
            self.scheduler.schedule_later(
                30, 
                afk_channel.id, 
                afk_channel.delete(reason="Deleting temp mute channel.")
            )
```
versus
```py
            coro = afk_channel.delete(reason="Deleting temp mute channel.")
            self.scheduler.schedule_later(30, afk_channel.id, coro)
```